atlassian/bitbucket/cloud/base.py

```diff
@@ -58,15 +59,7 @@ def _get_paged(self, url, params=None, data=None, flags=None, trailing=None, abs
             if "values" not in response:
                 return
 
-            values = response.get("values", [])
-            if not response.get("size", -1) == len(values):
-                raise AssertionError(
-                    "Wrong response for url [{}], the size attribute doesn't match the number of recieved values:\n{}".format(
-                        self.url, response
-                    )
-                )
-
-            for value in values:
+            for value in response.get("values", []):
```

Comment:
There is #653 to fix this issue.

Reply:
Sure, had added this for my tests and get it to work. False commited.